lib/keyring_liberator.rb

```diff
@@ -21,12 +21,16 @@ def self.get_keyring(path)
     end
 
     def self.get_keyring_named(name)
-      get_all_keyrings.find { |k| k.name == name }
+      get_all_keyrings.find { |k| k.name == name && k.path == Dir.getwd }
```

Comment:
This line is different, though, and I might just remove it.  I don't like that it adds a check against `Dir.getwd` (if I refactored, I would add a `path` argument to compare against).